src/IdBrokerClient.php

```diff
@@ -312,16 +312,18 @@ public function listUsers(array $fields = null, ?array $search = []): array
      * @param string $type
      * @param string|null $label
      * @param string|null $rpOrigin
+     * @param string|null $recovery_email
      * @return array|null
      * @throws ServiceException
      */
-    public function mfaCreate(string $employee_id, string $type, string $label = null, string $rpOrigin = ''): ?array
+    public function mfaCreate(string $employee_id, string $type, string $label = null, string $rpOrigin = '', string $recovery_email = null): ?array
```

Comment:
Strings in php are not nullable by default. (Also can fix the label while you are at it.)

```suggestion
    public function mfaCreate(string $employee_id, string $type, ?string $label = null, string $rpOrigin = '', ?string $recovery_email = ''): ?array
```